src/widgets/hierarchical-menu/hierarchical-menu.js

```diff
@@ -46,7 +46,7 @@ hierarchicalMenu({
   [ separator=' > ' ],
   [ rootPath ],
   [ showParentLevel=true ],
-  [ limit=1000 ],
+  [ limit=30 ],
```

Comment:
I thought 30 was a good "this is a default value that will show you a lot but you should configure it"